### ISSUE TITLE
elearning module it does not allow me to add an html option

### ISSUE THREAD

**Issue by Juankpoyo** [odoo/odoo#57220]
Impacted versions:
Steps to reproduce/add html custom

Current behavior:

Expected behavior:
In the elearning module it does not allow me to add an html option, when it is entered automatically it is eliminated,

Now the html option if it works in all other parts of the website is only Elearning.

Video/Screenshot link (optional):

![Captura de Pantalla 2020-09-07 a la(s) 18 11 31](https://user-images.githubusercontent.com/70921910/92421400-96930680-f135-11ea-8c28-4b8c9b505a04.png)

![Captura de Pantalla 2020-09-07 a la(s) 18 12 56](https://user-images.githubusercontent.com/70921910/92421463-d4902a80-f135-11ea-89da-90962d239578.png)

**Comment by pedrobaeza** [comment]
How do you get to that HTML insertion?

**Comment by Yenthe666** [comment]
@pedrobaeza he clicks on "Edit HTML" at the left with Studio :)

**Comment by pedrobaeza** [comment]
Ouch, but that is something not supported AFAIK. Odoo Studio is not for that modifications.

**Comment by Yenthe666** [comment]
Yeah I'd kind off like to label this as non-support / out of scope. Plus iframes are usually not a pretty idea either.
